static/src/javascripts/projects/common/modules/experiments/tests/measure-understanding.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import { addEventListener } from 'lib/events';
+import config from 'lib/config';
+import fastdom from 'lib/fastdom-promise';
+import mediator from 'lib/mediator';
+import { local as localStorage } from 'lib/storage';
+import template from 'lodash/utilities/template';
+import ophan from 'ophan/ng';
+import measureUnderstandingStr
+    from 'raw-loader!common/views/experiments/measure-understanding.html';
+import { markup as thumb } from 'svg-loader!svgs/icon/thumb.svg';
+
+const MeasureUnderstanding = () => {
+    // Test id
+    const id = 'MeasureUnderstanding';
+
+    // Ophan survey id
+    const component = 'measure_understanding';
+
+    // Test duration
+    const start = '2017-04-27';
+    const expiry = '2017-05-27';
+
+    // will run in articles only if no story questions atom is already there
+    // *and* the user hasn't already answered the question
+    const canRun = () =>
+        config.page.contentType === 'Article' &&
+        !config.page.isAdvertisementFeature &&
+        !localStorage.get(id) &&
+        !document.getElementsByClassName('js-view-tracking-component').length;
```

Comment:
Nope, this is to make sure that we're not adding the component if there already is a story questions package at the bottom of the article.